dockerfiles/init/modules/keycloak/files/che/login/register.ftl

```diff
@@ -0,0 +1,76 @@
+<#import "template.ftl" as layout>
```

Comment:
I am not sure, whether I understood you correctly, but if you meant to roll our changes over base templates - I do not think that it is a good idea here. For example, they could redesign some pages, do we want to apply that? I think no.